automation/check-patch.e2e-okd.sh

```diff
@@ -23,6 +23,9 @@ main() {
     # Let's fail fast if it's not compiling
     make handler
 
+    # The OCP provider requires jq
+    sudo yum install -y jq
```

Comment:
dnf or yum ? or both available i guess ?

Reply:
also do we have docker-ce latest build ? for big images
(i guess, lets see , or add docker --version print so we know if stuff happens)

Reply:
actually, let me add it to the job. i don't want to install deps here. this script should just run tests on any environment (with deps in place)